utils/utils.tsx

```diff
@@ -1342,15 +1350,15 @@ export function fillArray(value, length) {
 
 // Checks if a data transfer contains files not text, folders, etc..
 // Slightly modified from http://stackoverflow.com/questions/6848043/how-do-i-detect-a-file-is-being-dragged-rather-than-a-draggable-element-on-my-pa
-export function isFileTransfer(files) {
+export function isFileTransfer(files: any) {
```

Comment:
Could this use the built-in `FileList` type?

Reply:
looked like it was a `DataTransfer`